packages/discovery-react-components/src/components/DocumentPreview/components/HtmlView/HtmlView.tsx

```diff
@@ -60,6 +60,18 @@ export const HtmlView: FC<Props> = ({
   const [processedDoc, setProcessedDoc] = useState<ProcessedDoc | null>(null);
   const [highlightLocations, setHighlightLocations] = useState<Location[]>([]);
 
+  useEffect(() => {
```

Comment:
do you want to use https://reactjs.org/docs/hooks-reference.html#uselayouteffect here? i suppose `DOMPurify` handles rerendering just fine, but i thought `useLayoutEffect` was used exactly for this purpose (after rendering but before painting to make last minute modifications)

Reply:
Good point, i will make that change. Thanks!